docs/scaffold.md

```diff
@@ -0,0 +1,28 @@
+# Start project
+
+## Overview
+
+To see how easy it is to use Bigflow you can use `start-project` which creates a sample standalone project with a [standard structure](https://github.com/allegro/bigflow/blob/master/docs/project_setup_and_build.md#project-structure).
+This project contains an Apache Beam workflow and a BigQuery workflow.
+You will be able to deploy those workflows in your GCP project providing informations like project_id/bucket etc.
```

Comment:
You will be able to deploy those workflows on your GCP project providing information like project_id/bucket etc.